apps/menu-bar/src/popover/Core.tsx

```diff
@@ -201,6 +203,13 @@ function Core(props: Props) {
 
   const handleUpdateUrl = useCallback(
     async (url: string) => {
+      if (!storage.getString(sessionSecretStorageKey)) {
+        Alert.alert(
+          'You need to be logged in to launch updates.',
+          'Please log in through the Settings window and try again.'
```

Comment:
```suggestion
          'Log in through the Settings window and try again.'
```

avoid saying please, it's unnecessary / adds noise